src/main/java/task/TaskManager.java

```diff
@@ -0,0 +1,136 @@
+package task;
+
+import error.exception.DukeEmptyTaskDescriptionException;
+import error.exception.DukeInvalidDescriptionFormatException;
+import task.subtask.Task;
+import task.subtask.Todo;
+import task.subtask.Deadline;
+import task.subtask.Event;
+
+import java.util.Scanner;
+
+public class TaskManager extends Duke {
+    private static final int INDEX_AFTER_TODO = 4;
+    private static final int INDEX_AFTER_DEADLINE = 8;
+    private static final int INDEX_AFTER_EVENT = 5;
+
+    /**
+     * Extracts the description, creates a todo Task object and stores the task in the list
+     *
+     * @param input is the command given by the user
+     * @return the todo object created under Task
+     * @throws DukeEmptyTaskDescriptionException if no description is present after todo command
+     */
```

Comment:
Great use of the java documentation here! The comments make your methods simple and easy to understand!